providers/stores/azblobHarvestStore.js

```diff
@@ -85,6 +91,35 @@ class AzHarvestBlobStore extends AbstractAzBlobStore {
       }, {})
     })
   }
+
+  /**
+   * Get the latest version of each tool output for the given coordinates. The coordinates must be all the way down
+   * to a revision.
+   * @param {EntityCoordinates} coordinates - The component revision to report on
+   * @returns {Promise} A promise that resolves to an object with a property for each tool and tool version
+   *
+   */
+  async getAllLatest(coordinates) {
+    const allFiles = await this._getAllFiles(coordinates)
```

Comment:
Suggesting name change here and in fileHarvestStore to: 
```suggestion
    const allFilesList = await this._getListOfAllFiles(coordinates)
```